rpcq/_base.py

```diff
@@ -144,7 +135,8 @@ def _default(obj):
 def _object_hook(obj):
     if '_type' in obj:
         try:
-            msg_type = Message.types()[obj["_type"]]
+            import pidgin.core_messages
```

Comment:
This particular `pidgin` instance makes me wonder whether this will actually work with RPCQ extensions. This comment is a reminder to myself to investigate (and probably fix this function).